prototypes/oxygen-diffuser.lua

```diff
@@ -193,7 +193,8 @@ data:extend{
     --damaged_trigger_effect = hit_effects.entity(),
     fluid_box =
     {
-      base_area = 1,
+      base_area = 2,
+      base_level = -1,
```

Comment:
What does this change do?

Reply:
[FluidBox docs](https://lua-api.factorio.com/latest/types/FluidBox.html>

All FB are at level 0. Putting it at -1 means that fluids generally "fall" into it. Which means that the oxygenator simply sucks oxygen from the pipes nearby, instead of equally sharing it.
This was not strictly necessary, but since fluids are deleted via script, if the pipe is half empty (say 20 fluid out of 100%) and it needed 24, it couldn't get any.
 So now it basically has priority. It will buffer 1x2x100 fluid = 200 oxygen (height x base x 100). The buffer increase was not strictly required, but helps since updates are every 60 ticks instead of every tick (8 machines are 24 oxygen/s right now), so 200 is something short of 10s of operability